vision/standard_object/odlc_image_processing.py

```diff
@@ -0,0 +1,67 @@
+"""
+Preprocesses images obtained during airdrop area flyover for
+use in contour detection for the standard objects.
+"""
+
+import cv2
+import numpy as np
+
+from vision.common.constants import Image, Kernel
+
+
+def preprocess_std_odlc(image: Image, thresh_min: int, thresh_max: int) -> Image:
+    """
+    Preprocesses image for use in detecting the contours of standard objects.
+
+    Parameters
+    ----------
+    image : Image
+        image from airdrop area flyover before any processing has occured
+    thresh_min: int
+        The minimum threshold input for the Canny edge detection
+    thresh_max: int
+        The maximum threshold input for the Canny edge detection
```

Comment:
These 2 parameters should have default values so there is a clear idea of the kind of values that this function would be expecting.